sszgen/main.go

```diff
@@ -1030,11 +1043,15 @@ func (e *env) parseASTFieldType(name, tags string, expr ast.Expr) (*Value, error
 			return &Value{t: TypeBitList, m: maxSize, s: maxSize}, nil
 		} else if strings.HasPrefix(sel, "Bitvector") {
 			// go-bitfield/Bitvector, fixed bytes
-			size, ok := getTagsInt(tags, "ssz-size")
-			if !ok {
-				return nil, fmt.Errorf("bitvector %s does not have ssz-size tag", name)
+			dims, err := extractSSZDimensions(tags)
+			if err != nil {
+				return nil, fmt.Errorf("failed to parse ssz-size tag for bitvector %s, err=%s", name, err)
 			}
-			return &Value{t: TypeBytes, s: size, n: size}, nil
+			tailDim := dims[len(dims)-1] // get last value in case this value is nested within a List/Vector
```

Comment:
any chance len dims could be 0?

Reply:
shouldn't be, since a Bitvector should always be a slice of some kind w/ tags, but I can add a defensive check so there's a readable error in that case.